src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -371,6 +371,38 @@ prefetch('/about');
 
 <ReadMore>Learn more about [prefetching](/en/guides/prefetch/).</ReadMore>
 
+### Removed: `rewrite()` from Actions context
+
+<SourcePR number="14477" title="feat!: remove rewrite from action context"/>
+
+In Astro 5.5.6, `rewrite()` on Actions context was deprecated as it was not meant to be exposed here.
+
+Astro 6.0 removes this property entirely on Actions context.
+
+#### What should I do?
+
+Review your Actions handlers and remove any call to `rewrite()`:
+
+
+```ts title="src/actions/index.ts" del={10}
+import { defineAction } from 'astro:actions';
+import { z } from 'astro:schema';
+
+export const server = {
+  getGreeting: defineAction({
+    input: z.object({
+      // ...
+    }),
+    handler: async (input, context) => {
+      context.rewrite('/')
+      // ...
+    }
+  })
+}
+```
+
+<ReadMore>Learn more about [rewrites](/en/guides/routing/#rewrites).</ReadMore>
```

Comment:
I would only use rewrites as the link here if we actually *discuss/explain* the rewrite concept in the deprecation method above, and we think this really is informing someone more about redirects and not about actions.

In fact, this probably goes to https://docs.astro.build/en/reference/modules/astro-actions/#handler-property, which I notice describes this "mostly full context object" (so yeah, I think this is the good link?) BUT says it excludes *redirect*, but not rewrite! So, can we double check these reference docs too?

Reply:
Just checking, we're linking to rewrites or to handling actions?